src/core/lib/surface/init_internally.h

```diff
@@ -26,10 +26,21 @@ extern bool (*IsInitializedInternally)();
 
 class KeepsGrpcInitialized {
  public:
-  KeepsGrpcInitialized() { InitInternally(); }
-  ~KeepsGrpcInitialized() { ShutdownInternally(); }
+  explicit KeepsGrpcInitialized(bool enabled) : enabled_(enabled) {
```

Comment:
Suggest making this parameter default to true, so that you don't need to touch any of the existing callers.

Reply:
Done